replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicatorConfigImpl.java

```diff
@@ -63,6 +74,8 @@
 
   private boolean suspended;
 
+  private Date modified;
```

Comment:
✏️ I would suggest moving this attribute as part of the base class for all persistable